### PR TITLE
Updating configs of odo to reflect its move into openshift org

### PR DESCRIPTION
**Pull request by mohammedzee1000**
- Updating cluster prow config and plugin
 - Moving config files to openshift/odo
 - Renaming files from *redhat-developer* to *openshift*

Signed-off-by: Mohammed Zeeshan Ahmed <mohammed.zee1000@gmail.com>

/hold
/assign @kadel

### CODE REVIEW

**Comment by mohammedzee1000** [comment]
If ci fails after paths fix, may need https://github.com/openshift/odo/pull/1502

**Comment by mohammedzee1000** [comment]
/assign @bbguimaraes
It would be nice if the config can be validated. We are working on getting tests themselves to pass :)

File: ci-operator/jobs/openshift/odo/openshift-odo-master-presubmits.yaml [hunk 1]
```diff
@@ -26,12 +26,12 @@ presubmits:
...
           valueFrom:
             configMapKeyRef:
-              key: redhat-developer-odo-master.yaml
+              key: openshift-odo-master.yaml
               name: ci-operator-master-configs
         - name: JOB_NAME_SAFE
           value: e2e-core
         - name: TEST_COMMAND
-          value: |-
+          value: |
```
Comment (stevekuznetsov) [inline ci-operator/jobs/openshift/odo/openshift-odo-master-presubmits.yaml]: interesting

Comment (mohammedzee1000) [reply ci-operator/jobs/openshift/odo/openshift-odo-master-presubmits.yaml]: Yes it is. I wondered what happened there as these files are generated and not edited by hand

**Pull request merged**